src/shared-components/alert/style.ts

```diff
@@ -50,20 +50,23 @@ const errorAlertStyles = css`
   box-shadow: 0px 8px 24px rgba(189, 32, 15, 0.05);
 `;
 
-export const AlertContainer = styled.div`
+export const AlertContainer = styled.div<{
+  exiting: boolean;
+  alertType: string;
```

Comment:
It might be preferable to export `type AlertType = 'success' | 'error' | 'default' used both in the props of the component and also here, to be more specific than `string`.